documentation/Functions.md

```diff
@@ -240,11 +241,27 @@ image: Path to an image to be displayed in the katacoda step.
 
 ***
 
-
 ### adaptTemplatesCobiGen
 #### parameter
 * No parameters
 #### example
 adaptTemplatesCobiGen()
 
 ***
+
+### createDevon4ngProject
+#### parameter 
+1. Name of the Project.
+2. Path to where the Project should be created (relative to workspace). Folder should exist.
```

Comment:
Here you should mention that the path can be an empty string and that the project then will be created in the workspace directory.